src/test/context.test.ts

```diff
@@ -57,7 +57,7 @@ class TestContext
     {
         try
         {
-            fs.unlinkSync(join(this.root, '.vscode', '.rock.json'));
```

Comment:
This is a hidden file because the user is not supposed to edit this by hand (there is a UI for that). If we want to let/expect the user to edit it (we shouldn't IMO because it is not intuitive, setting a debugging target for example requires creating an object) we should move it to a proper configuration provided by the extension so the user will at least get a description of what each field means.

Reply:
Also, vscode provides an API to manage configuration if we decide to make this visible to the user...

Reply:
Actually, I just saw that there is a place where the extensions are supposed to save their private stuff. I'll check it out.

Reply:
I prefered this because it was easier to keep it leak-free but what you are referring to is definitely a valid alternative.